Documentation/learning/api_guarantees.md

```diff
@@ -31,36 +31,23 @@ An etcd operation that modifies the key value store is assigned a single increas
 
 All API requests are atomic; an operation either completes entirely or not at all. For watch requests, all events generated by one operation will be in one watch response. Watch never observes partial events for a single operation.
 
-#### Consistency
-
-All API calls ensure [sequential consistency][seq_consistency], the strongest consistency guarantee available from distributed systems. No matter which etcd member server a client makes requests to, a client reads the same events in the same order. If two members complete the same number of operations, the state of the two members is consistent.
-
-For watch operations, etcd guarantees to return the same value for the same key across all members for the same revision. For range operations, etcd has a similar guarantee for [linearized][Linearizability] access; serialized access may be behind the quorum state, so that the later revision is not yet available.
-
-As with all distributed systems, it is impossible for etcd to ensure [strict consistency][strict_consistency]. etcd does not guarantee that it will return to a read the “most recent” value (as measured by a wall clock when a request is completed) available on any cluster member.
-
-#### Isolation
-
-etcd ensures [serializable isolation][serializable_isolation], which is the highest isolation level available in distributed systems. Read operations will never observe any intermediate data.
-
 #### Durability
 
 Any completed operations are durable. All accessible data is also durable data. A read will never return data that has not been made durable.
 
-#### Linearizability
+#### Isolation level and consistency of replicas
+
+etcd ensures [serializable isolation][serializable_isolation], which is the highest isolation level available in transactional database systems. Read operations will never observe any intermediate data.
 
-Linearizability (also known as Atomic Consistency or External Consistency) is a consistency level between strict consistency and sequential consistency. 
+etcd ensures linearizability as consistency of replicas basically. As described below, exceptions are watch operations and read operations which explicitly specifies serializable option.
```

Comment:
we can mention that etcd satisfy strict serializable. @aphyr do you know if there is a good definition on `strict serializable` that we can link to?

Reply:
yeah, I'll add the link (http://jepsen.io/consistency/models/strict-serializable) provided by @aphyr in addition to Herlihy & Wing